pkg/commands/copy.go

```diff
@@ -197,21 +197,43 @@ func (cr *CachingCopyCommand) From() string {
 }
 
 func resolveIfSymlink(destPath string) (string, error) {
-	baseDir := filepath.Dir(destPath)
-	if info, err := os.Lstat(baseDir); err == nil {
+	if !filepath.IsAbs(destPath) {
```

Comment:
Why is this check needed?

Reply:
If it is rel path, it is not return value of `util.DestinationFilepath()`. Would it be correct to append the working directory to destPath in this function?